jekyll-exporter.php

```diff
@@ -149,28 +274,43 @@ function get_posts() {
 	function convert_meta( $post ) {
 
 		$output = array(
-			'id'      => $post->ID,
+			'post_id'      => $post->ID,
```

Comment:
Will this be a breaking change?

Reply:
Missed that also. Reverted and put this as a configurable option.